### PR TITLE
initial hello world -cucumber format

### PR DESCRIPTION
**Pull request by Sara-raj**
image

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -3,11 +3,15 @@ import sbt._
...
+    "io.cucumber"       %% "cucumber-scala"         % "6.9.1"  % Test,
+    "io.cucumber"        % "cucumber-junit"         % "6.9.1"  % Test,
+    "junit"              % "junit"                  % "4.12"   % Test,
+    "com.novocode"       % "junit-interface"        % "0.11"   % Test,
+    "com.typesafe"       % "config"                 % "1.3.2"  % Test,
+    "com.typesafe.play" %% "play-ahc-ws-standalone" % "2.1.2"  % Test,
+    "org.slf4j"          % "slf4j-simple"           % "1.7.25" % Test,
+    "org.julienrf"      %% "play-json-derived-codecs"  % "7.0.0",Test,
+    "com.beachape"       %% "enumeratum-play-json"    % "1.6.1",Test
   )
```
Comment (girishkamat) [inline project/Dependencies.scala:16]: Formatting could be improved. We use scalafmt plugin

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -17,41 +17,9 @@ environment: local
...
-  services {
-    host: ""
-    auth {
-      port = 8585
-      productionRoute = ""
-    }
-    ims {
-      port = 9653
+    marginal-relief-calculator {
+      port = 7055
```
Comment (girishkamat) [inline src/test/resources/application.conf:21]: Shouldnt this port be 7100 as we discussed?

File: src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmrc.test.api.requests
```
Comment (girishkamat) [inline src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala:1]: header missing

File: src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  //Used by hello world only
+  def getMarginalReliefCalculatorRequests(endpoint: String): StandaloneWSResponse = {
+
+    val baseUri = s"$marginalreliefCalculatorApiUrl/marginal-relief-calculator$endpoint"
+    val headers = Map(
+      "Content-Type" -> "application/json",
+      "Accept"       -> "application/vnd.hmrc.1.0+json"
+    )
+
+    print("Marginal Relief Calculator baseUri ************************" + baseUri)
```
Comment (girishkamat) [inline src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala:18]: println?

Comment (Sara-raj) [reply src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala:18]: yes this is to help testers to see the baseUri info which is more helpful when it comes to debugging.

Comment (girishkamat) [reply src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala:18]: Can we not use loggers instead?

File: project/Dependencies.scala (hunk 1 above)
Comment (Sara-raj) [reply project/Dependencies.scala:16]: Ah. this is an auto-generated file. Cool i got the scalafmt plugin (cmd+option+l) done to fix the format. Hope this is fixed now.

Comment (girishkamat) [reply project/Dependencies.scala:16]: You can add the following alias to build.sbt and do it all in one shot

`addCommandAlias("fmt", "scalafmt;scalafmtSbt;test:scalafmt;it:scalafmt")
`

File: .bsp/sbt.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.8","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_321.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/sarakarthik/Library/Application Support/JetBrains/IdeaIC2021.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/sarakarthik/Library/Application%20Support/JetBrains/IdeaIC2021.3/plugins/Scala/launcher/sbt-launch.jar"]}
```
[the remark this answers is not in the document]
Comment (Sara-raj) [reply .bsp/sbt.json:1]: added to .gitignore.

File: src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala (hunk 4 above)
Comment (Sara-raj) [reply src/test/scala/uk/gov/hmrc/test/api/requests/MarginalReliefCalculatorRequests.scala:18]: Actually loggers does the job, this empty println are just to add a space between request and response(jus to read the console error). Checked across many tests and spotted this empty println :( Pls confirm if its Okay ?

File: .bsp/sbt.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (girishkamat) [reply .bsp/sbt.json:1]: Can we delete this file from the PR?

**Review by girishkamat — APPROVED** [review]

**Pull request merged**